s3/pkg/meta/db/drivers/tidb/object.go

```diff
@@ -144,17 +93,8 @@ func (t *TidbClient) PutObject(ctx context.Context, object *Object, tx interface
 	sqlTx, _ = tx.(*sql.Tx)
 	sql, args := object.GetCreateSql()
 	_, err = sqlTx.Exec(sql, args...)
-	/*if object.Parts != nil {
-		v := math.MaxUint64 - uint64(object.LastModified)
-		version := strconv.FormatUint(v, 10)
-		for _, p := range object.Parts {
-			psql, args := p.GetCreateSql(object.BucketName, object.ObjectKey, version)
-			_, err = sqlTx.Exec(psql, args...)
-			if err != nil {
-				return err
-			}
-		}
-	}*/
+	// TODO: multi-part handle
```

Comment:
please create issue and update the TODO comment with who owns theTODO, thanks

Reply:
Hi we already have the issue[#690](https://github.com/opensds/multi-cloud/issues/690), and I added the link of this issue to the TODO comment.